packages/functional-tests/src/tests/button-targeting-test.ts

```diff
@@ -0,0 +1,177 @@
+/*!
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import * as MRE from '@microsoft/mixed-reality-extension-sdk';
+
+import { Test } from '../test';
+import { Vector3Like } from '@microsoft/mixed-reality-extension-sdk';
```

Comment:
Remove import of Vector3Like, and reference it as MRE.Vector3Like.

Reply:
Wish VS Code auto importing would figure this out.

---
In reply to: [428198599](https://github.com/microsoft/mixed-reality-extension-sdk/pull/593#discussion_r428198599) [](ancestors = 428198599)